packages/web-pkg/src/composables/piniaStores/theme.ts

```diff
@@ -94,19 +93,29 @@ export const useThemeStore = defineStore('theme', () => {
   const initializeThemes = (themeConfig: WebThemeConfigType) => {
     availableThemes.value = themeConfig.themes.map((theme) => merge(themeConfig.defaults, theme))
 
-    if (unref(currentThemeName) === null) {
-      currentThemeName.value = unref(availableThemes).find((t) => t.isDark === unref(isDark)).name
-    }
-
+    const firstLightTheme = unref(availableThemes).find((theme) => !theme.isDark)
+    const firstDarkTheme = unref(availableThemes).find((theme) => theme.isDark)
     setAndApplyTheme(
-      unref(availableThemes).find((t) => t.name === unref(currentThemeName)) ||
-        availableThemes.value[0]
+      unref(availableThemes).find((t) => t.name === unref(currentLocalStorageThemeName)) ||
+        (unref(isDark) ? firstDarkTheme : firstLightTheme),
+      false
     )
   }
 
-  const setAndApplyTheme = (theme: WebThemeType) => {
+  const setAutoSystemTheme = () => {
+    currentLocalStorageThemeName.value = null
+    setAndApplyTheme(unref(isDark) ? availableThemes.value[1] : availableThemes.value[0], false)
```

Comment:
Same issue here, please find the first light and dark theme. Check if happy path is needed